packages/clay-drop-down/src/Menu.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import {Align} from 'metal-position';
+import {Portal} from '@clayui/shared';
+import {useDropdownCloseInteractions} from './hooks';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Flag to indicate if menu is showing or not.
+	 */
+	active?: boolean;
+
+	/**
+	 * HTML element that the menu should be aligned to
+	 */
+	alignElementRef: React.RefObject<HTMLElement>;
+
+	/**
+	 * Default position of menu element. Values come from `metal-position`.
+	 *
+	 * Align.TopCenter = 0;
+	 * Align.TopRight = 1;
+	 * Align.RightCenter = 2;
+	 * Align.BottomRight = 3;
+	 * Align.BottomCenter = 4;
+	 * Align.BottomLeft = 5;
+	 * Align.LeftCenter = 6;
+	 * Align.TopLeft = 7;
+	 *
+	 * Defaults to BottomLEft
+	 */
+	alignmentPosition?: number;
```

Comment:
Would not it be better to expose an enum? so you do not need to export `Align` to pass the correct information and can set the best type. What do you think?

```typescript
enum AlignmentPosition {
  TopCenter = 0,
  TopRight = 1,
  RightCenter = 2,
  BottomRight = 3,
...
}
```

Reply:
I went directly with the `Align` export from metal-postion because that is the consumer of that prop. I didn't want to create another definition for it and would rather just pass metal-position's align straight through.

Reply:
No problem, I think in this way the experience improves who is using Typescript and React for having a full type definition and if we change it in the future we would not have problems with export.